README.md

```diff
@@ -343,6 +362,54 @@ some line inside the exported test function). You can run just that test with:
 teenytest test/bar-test.js:14
 ```
 
+#### Locating with multiple names or line numbers
+
+Each `testLocator` option can include one name or line number filter suffix.
+The same glob may be passed multiple times with different suffixes to locate
+tests matching more than one filter:
+
+```
+teenytest \
+  test/foo-test.js#red \
+  test/foo-test.js#blue \
+  test/bar-test.js:14 \
+  test/bar-test.js:28
+```
+
+The above will run tests named `red` and `blue` in the file `test/foo-test.js`
+and tests on lines 14 and 28 in the file `test/bar-test.js`.
+
+#### Locating with the `--example` option
```

Comment:
I had to re-read the PR a couple times to understand what was meant by `--example`, assuming this is a BDD/rspecism that leaked from the code. Would `--name` be more predictable and consistent with other tools like minitest?

If you agree, I'd ask you to change the name of this option to `--name` or similar before we merge

Reply:
@searls This has been changed to `--name`